enhancements/observability/multicluster-observability-addon.md

```diff
@@ -0,0 +1,649 @@
+          - application
+          name: application-logs
+          outputRefs:
+          - application-logs
+        - inputRefs:
+          - audit
+          - infrastructure
+          name: cluster-logs
+          outputRefs:
+          - cluster-logs
+```
+
+#### Multi Cluster OTLP Collection and Forwarding
+
+For all managed clusters the fleet administrator is required to provide, at least, a single `OpenTelemetryCollector` resource stanza that describes the OpenTelemetry forwarding configuration for the entire fleet in the default namespace `open-cluster-management`.
+
+The following example resource describes a configuration where a `OpenTelemetryCollector` instance is deployed per spoke cluster and it sends data to a Hub OTEL Cluster (note that this cluster can be different from the RHACM Hub cluster). The Hub OTEL Cluster then exports the received telemetry to a third-party storage solution.
+
+```mermaid
+flowchart TD
+    subgraph Spoke cluster
+    A[OTEL Collector] 
+    end
+    subgraph Spoke cluster
+    B[OTEL Collector] 
+    end
+    subgraph Spoke cluster
+    C[OTEL Collector] 
+    end
+    subgraph Hub OTEL Cluster
+    D[OTEL Collector] 
+    E[Grafana Tempo]
+    end
+    A --> |otlp|D
+    B --> |otlp|D
+    C --> |otlp|D
+    D --> E
+    subgraph Third party services
+    D --> F[Dynatrace]
+    D --> G[Datadog]
+    end
+```
+
+
+The following example resource describes a configuration for forwarding application otlp data from one OpenTelemetry Collector (deployed in the spoke cluster) to the OpenTelemetry Hub instance:
+
+```yaml
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: spoke-otelcol
+  namespace: open-cluster-management
+spec:
+  config: |
+    receivers:
+      jaeger:
+        protocols:
+          grpc:
+      otlp:
+        protocols:
+          grpc:
+          http:
+    processors:
+    exporters:
+      debug:
+      otlp:
+        tls:
+          insecure: true
+    service:
+      pipelines:
+        traces:
+          receivers: [jaeger,otlp]
+          processors: []
+          exporters: [otlp, debug]
+```
+
+For the `otlp` exporter MCOA will provide a Secret `tracing-otlp-auth` with all required authentication information, e.g. TLS client Certificate:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: tracing-otlp-auth
+  namespace: spoke
+stringData:
+  'tls.crt': "TLS client certificate"
+  'tls.key': "TLS key"
+  'ca-bundle.crt': "Certificate Authority certificate" # Pull from the referenced Secret otel-gateway
+```
+
+In turn the addon will compile a `ManifestWork` for the managed cluster `spoke` as follows and pass it over it's WorkAgentController:
+
+```yaml
+kind: ManifestWork
+metadata:
+  name: addon-multicluster-observability-addon-deploy-0
+  namespace: spoke
+spec:
+  workload:
+    manifests:
+    - apiVersion: project.openshift.io/v1
+      kind: Project
+      metadata:
+        name: openshift-opentelemetry-operator
```

Comment:
Aligning this with the current charts in [here](https://github.com/rhobs/multicluster-observability-addon/tree/main/internal/addon/manifests/charts/mcoa/charts/tracing/templates) (cc @iblancasa):

```suggestion
    - apiVersion: v1
      kind: Namespace
      metadata:
        name: openshift-opentelemetry-operator
    - apiVersion: v1
      kind: Namespace
      metadata:
        name: mcoa-opentelemetry
```